src/__tests__/initRules.test.js

```diff
@@ -1584,13 +1598,11 @@ describe('initRules', function() {
         'does execute the next action in the chain if any action takes' +
           ' longer than 2 seconds to complete',
```

Comment:
I think this test should say "does not execute" now.

Reply:
Done.